clinics/merge/clinic.go

```diff
@@ -198,25 +200,27 @@ func (m *ClinicMergePlanner) CliniciansMergePlan(ctx context.Context, source, ta
 
 func (m *ClinicMergePlanner) listAllPatients(ctx context.Context, clinic clinics.Clinic) ([]patients.Patient, error) {
 	clinicId := clinic.Id.Hex()
-	limit := 1000000
 
+	filter := patients.Filter{ClinicId: &clinicId, ExcludeSummary: true}
```

Comment:
Nit: for the purpose of merging, wouldn't we also want `ExcludeDemo: true`?

Reply:
Since I was making more tweaks I went ahead and added this.